devicon.json

```
@@ -5776,6 +5776,27 @@
         },
         "color": "#294172", 
         "aliases": [] 
-    }
+    },
+{
+        "name": "solidjs",
```

Add empty altnames array. You can fill it with alternative names if there are any
```suggestion
        "name": "solidjs",
        "altnames": [],
```